spec/factories/tasks.rb

```diff
@@ -5,5 +5,11 @@
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
+    association :project
+
+    trait :status_done_yesterday_completion_date do
```

Comment:
ステータスが完了になると完了日も登録される仕様であることは読み取れると思うので、trait名で完了日付について言及しなくても良いかと思いました。今回は`done`で十分伝わるかと思います。

テストケースがもっと多く、タスクの完了日によってテストデータを使い分けている場合は`done_yesterday`という形で定義すれば良いかと思います。（他のtraitは done_2_week_ago のように記載する想定です。）
```suggestion
    trait :done do
```

Reply:
1bc712b でtrait名の修正致しました。

なるほど。。テストデータの種類や量でも変わってくるんですね。
ありがとうございます！

Reply:
> テストデータの種類や量でも変わってくる

全体のテストケースを考慮した上で、どのような命名だと無駄なく適切に伝わるのか、という意識を持てばOKです！